builtin/logical/transit/path_cache_config.go

```diff
@@ -48,6 +48,12 @@ func (b *backend) pathCacheConfigWrite(ctx context.Context, req *logical.Request
 	if cacheSize < 0 {
 		return logical.ErrorResponse("size must be greater or equal to 0"), logical.ErrInvalidRequest
 	}
+    minCacheSize := 10
+    var resp = &logical.Response{}
+    if cacheSize != 0 && cacheSize < minCacheSize {
+		resp.AddWarning(fmt.Sprintf("size %d is less than default minimum %d. Cache size is set to %d", cacheSize, minCacheSize, minCacheSize))
```

Comment:
In the case of config, lets actually error out if they try to set a too small value.